cmds/xbmc.py

```diff
@@ -664,29 +675,27 @@ def _files(bot,args,dirs,pid):
   if len(matches)==0:
     return 'Found 0 matches'
 
-  # default to top dir if every match is a sub dir
-  matches = util.reducetree(matches)
-
-  if len(matches)>1:
+  if len(matches)>1 and playall==False:
     maxm = bot.opt('library.max_matches')
     if maxm<1 or len(matches)<=maxm:
       return 'Found '+str(len(matches))+' matches: '+util.list2str(matches)
     else:
       return 'Found '+str(len(matches))+' matches'
 
   # translate library path if necessary
-  match = bot.library_translate(matches[0])
+  #match = bot.library_translate(matches[0]) 
```

Comment:
Commenting this line without replacing it breaks; normally I wouldn't care that much, since I remember you saying you're planning on making library_translate() work on lists, but the single sibyl user outside our group of friends users library.remote so would prefer if you extended this at the same time <3